src/com/couchbase/perf/shared/main/main.groovy

```diff
@@ -126,12 +126,7 @@ class Execute {
                     implementationsToAdd.add(new PerfConfig.Implementation(implementation.language, version, null, sha))
                 }
                 else if (implementation.language == "Go") {
-                    def sha = GoVersions.getLatestSha()
-                    def allReleases = GoVersions.getAllReleases()
-                    def highest = ImplementationVersion.highest(allReleases)
-                    ctx.env.log("Found latest sha for Go: ${sha}")
-                    String version = highest.toString() + "-" + sha
-                    implementationsToAdd.add(new PerfConfig.Implementation(implementation.language, version, null, null))
+                    implementationsToAdd.add(new PerfConfig.Implementation(implementation.language, "master", null, null))
```

Comment:
Hmm, not sure this is going to work though.  We're just going to end up with loads of database results for version "master", but they'll all be pointing at different versions.